build_and_test.sh

```diff
@@ -55,7 +60,7 @@ build
 tests
 
 if [ -n "$TRAVIS" ] || [ -n "$CI" ]; then
-  if [ "$os" != "freebsd" ]; then
+  if [ "${ID:-linux}" = "ubuntu" ] || [ "${ID:-linux}" = "debian" ]; then
```

Comment:
This is where I'd like the alpine package installs to go, just to keep everything in the same place.

Reply:
yeah if we do not use plugin.